packages/zoe/test/unitTests/contracts/newSwap/test-newSwap-swap.js

```diff
@@ -420,6 +421,9 @@ test('newSwap doubleSwap', async t => {
     publicFacet,
   ).makeAddLiquidityInvitation();
 
+  const collectFeesInvitation2 = E(creatorFacet).makeCollectFeesInvitation();
+  await zoe.offer(collectFeesInvitation2, undefined, undefined);
+
```

Comment:
These lines were added while building a test, and inadvertently ended up in a merge before they could be removed. Please drop them.